.github/workflows/build-and-push.yml

```diff
@@ -7,55 +7,19 @@ on:
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
+    if: github.repository_owner == 'sclorg'
     strategy:
+      fail-fast: false
       matrix:
-        version: [14, 16]
+        dockerfile_path: [ "14", "16" ]
```

Comment:
And what bout minimal?
As far as I know, we supports `12 14 14-minimal 16 16-minimal`

Reply:
They have their own action, but I can check and compare, and add them here if possible.